samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -13,7 +16,27 @@ int main(int argc, char* argv[]) try {
     if (3 != argc) {
         throw std::runtime_error(std::string{"Usage "} + argv[0] + " <MODEL_DIR> <IMAGE_FILE>");
     }
-    ov::Tensor image = utils::load_image(argv[2]);
+
+    // multinomial or beam_search can be used as well
+    ov::genai::GenerationConfig generation_config = ov::genai::greedy();
+    // ov::genai::GenerationConfig generation_config = ov::genai::vlm_multinomial();
+    // ov::genai::GenerationConfig generation_config = ov::genai::beam_search();
+
+    ov::AnyMap properies{ov::genai::generation_config(generation_config)};
+
+    // streamer could be used with greedy and multinomial
+     // if num_return_sequences > 1 in case of multinomial, the streamer will use the output from the first sequence
+    if (generation_config.is_greedy_decoding() or generation_config.is_multinomial()) {
+        properies.insert(ov::genai::streamer(print_subword));
+    }
+
+    std::vector<ov::Tensor> images = utils::load_images(argv[2]);
+
+    if (images.empty())
+        throw std::runtime_error(std::string{"No one image found by path "} + argv[2]);
```

Comment:
can it be handled within `load_images` itself? We can rely on `load_images` to return at least one image